src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.header__nav-list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  text-transform: uppercase;
+}
+
+.header__nav-link {
+  position: relative;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  color: inherit;
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  @include animation-hover(color, $blue-accent);
```

Comment:
Ensure that the `animation-hover` mixin is properly defined in `src/styles/utils/mixins.scss` and that it is following the code style rules. If it's a custom mixin, it should be well-documented to make the codebase maintainable.